src/mod_bosh.erl

```diff
@@ -355,8 +366,10 @@ maybe_start_session_on_known_host(Req, Body) ->
     catch
         error:Reason ->
             %% It's here because something catch-y was here before
-            ?ERROR_MSG("issue=bosh_stop issue=undefined_condition reason=~p",
-                       [Reason]),
+            ?LOG_ERROR(#{
+                issue => bosh_stop,
+                issue=>undefined_condition,
```

Comment:
This will override the previous value :(

Reply:
nice catch